SmartTutor/Database/Scripts/all.sql

```diff
@@ -27,6 +28,8 @@ DELETE FROM public."ArrangeTaskContainers";
 DELETE FROM public."ArrangeTasks";
 DELETE FROM public."LearningObjects";
 DELETE FROM public."LearningObjectSummaries";
+-- TODO: Learning objects reference Node Progress so the following line had to be moved here
```

Comment:
Are these TODOs done?

Reply:
Yes, I will go through all of the changes again to make sure everything is done